src/test/java/org/tugraz/sysds/test/component/tensor/HeterogeneousTensorColumnTest.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2019 Graz University of Technology
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.tugraz.sysds.test.component.tensor;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.tugraz.sysds.common.Types.ValueType;
+import org.tugraz.sysds.runtime.data.HeterogTensor;
+import org.tugraz.sysds.runtime.data.HomogTensor;
+
+import java.util.Arrays;
+import java.util.stream.IntStream;
+import java.util.stream.LongStream;
+
+
+public class HeterogeneousTensorColumnTest {
```

Comment:
Please rename the tests and test methods according to the renamed class hierarchy